tests/bsim/bluetooth/host/l2cap/multilink_peripheral/src/central.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/att.h>
+#include <zephyr/bluetooth/l2cap.h>
+#include <zephyr/logging/log.h>
+
+#include "testlib/scan.h"
+#include "testlib/conn.h"
+
+#include "babblekit/flags.h"
+#include "babblekit/sync.h"
+#include "babblekit/testcase.h"
+
+/* local includes */
+#include "data.h"
+
+LOG_MODULE_REGISTER(central, CONFIG_APP_LOG_LEVEL);
+
+static int total_rx;
```

Comment:
Suggest to use an unsigned value for a counter (size_t, unsigned int, uint32_t, whatever)

Reply:
var is useless so I removed it :+1: